app/frontend/styles/provider/_recruitment_performance_report.scss

```diff
@@ -1,11 +1,16 @@
 .recruitment-performance-report-table {
+  &__wrapper {
+    padding-bottom: govuk-spacing(4);
```

Comment:
Have we considered raising this up a few levels? Rather than the table having padding we could wrap each section (numbered heading and content) in a [`<section>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) tag. 
We could then add a `margin-bottom` to each section.

Even as I type this, I'd prefer to see the "spacing" added above the `<section>` tags onto  whatever is wrapping the sections - either using a `display: grid; gap: 20px` or [the owl selector](https://salehmubashar.com/blog/the-owl-selector-in-css) `* + * {margin-block-end: 20px}`

A bit more [explanation here](https://dcyoung.dev/shorts/refactoring-css-composition) 👌

Reply:
When I talked to Lindsay about this, she said that we only wanted to add the breathing space the bottom of the table. So the sections where there isn't a table (eg, if all the data is nil, or in the first section), we wouldn't want the extra breathing space. 

So it really is table specific rather than section specific.